src/crawlers/cheerio_crawler.js

```diff
@@ -171,7 +171,29 @@ const CHEERIO_OPTIMIZED_AUTOSCALED_POOL_OPTIONS = {
  *   For more information, see the [documentation](https://docs.apify.com/proxy).
  * @property {HandleFailedRequest} [handleFailedRequestFunction]
  *   A function to handle requests that failed more than `option.maxRequestRetries` times.
- *
+ * @property {Array<Hook>} [preNavigationHooks]
+ *   Async functions that are sequentially evaluated before the navigation. Good for setting additional cookies
+ *   or browser properties before navigation. The function accepts two parameters, `crawlingContext` and `gotoOptions`,
```

Comment:
I wonder if `gotoOptions` is a good name here. In browsers the name of the navigation function is `page.goto` but here it's `requestAsBrowser`. `requestAsBrowserOptions` is kinda long, but it tells you very exactly what the options are. And no-one will type this anyway.